### DISCUSSION TITLE
Back up and restore Data

### DISCUSSION THREAD

**Discussion by blogdemoi** [Nachtzuster/BirdNET-Pi#252] [Q&A] [answered]
Hi,

I noticed there are new options in "System Controls": Backup and Restore Data. What kind of Data is it about? 

Thank you.

**Comment by alexbelgium** [comment, marked as answer]
Hi, here is Nachzuster's description : https://github.com/Nachtzuster/BirdNET-Pi?tab=readme-ov-file#backup-and-restore
It encompasses database, settings, and audio files so can be quite big

https://github.com/Nachtzuster/BirdNET-Pi/blob/e1c25b816595dd95cf2750cbec232dd5617f81b9/scripts/backup_data.sh#L165-L176

**Reply by blogdemoi** [reply]
Thank you.
